examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -146,6 +149,13 @@ static void InitServer(intptr_t context)
 
     // Application code should always be initialised after the initialisation of server.
     ApplicationInit();
+
+#if CONFIG_ENABLE_SNTP_TIME_SYNC
+    char ntpServerUrl[]              = "pool.ntp.org";
+    uint16_t mSyncNtpTimeIntervalDay = 1;
```

Comment:
this is not a member - it looks like a constant. Should be `kSyncNtp....` and not `m...`